client/src/containers/PostDashboard.tsx

```diff
@@ -28,7 +29,7 @@ function RecentPosts(props: {
   const [openedPost, setOpenedPost] = React.useState(false);
 
   const checkForPosts = React.useCallback(() => {
-    if (!openedPost) {
+    if (!openedPost && !props.openedCreate) {
```

Comment:
Remove `openedPost` I think its useless now that we have removed post dialog and replaced it with a router